test/integration/targets/win_raw/tasks/main.yml

```diff
@@ -17,7 +17,7 @@
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
 - name: run getmac
-  raw: getmac
+  raw: getmac; exit $LASTEXITCODE
```

Comment:
these probably shouldn't be added; they're masking an issue with the raw behavior that makes it sort of "noncompliant";  maybe add to the raw command behavior in PSRP instead...